src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  color: #3d4e61;
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
```

Comment:
always set font styles to the `html` tag
in `*` tag only `box-sizing` can be used, anything else is forbidden